resources/geocoding/en/358.txt

```diff
@@ -33,6 +48,20 @@
 35856|Kymi
 35857|Kymi
 35858|Kymi
-3586|Vaasa
-3588|Oulu
+35861|Vaasa
+35862|Vaasa
+35863|Vaasa
+35864|Vaasa
+35865|Vaasa
+35866|Vaasa
+35867|Vaasa
+35868|Vaasa
+35881|Oulu
+35882|Oulu
+35883|Oulu
+35884|Oulu
+35885|Oulu
+35886|Oulu
+35887|Oulu
+35888|Oulu
 3589|Helsinki
```

Comment:
OK, didn't do that since 90 and 99 aren't in current use, but you are right as that might change in the future...